v0.5/classification_and_detection/run_common.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 if [ $# -lt 1 ]; then
-    echo "usage: $0 tf|onnxruntime|pytorch|tflite [resnet50|mobilenet|ssd-mobilenet|ssd-resnet34] [cpu|gpu]"
+    echo "usage: $0 tf|onnxruntime|pytorch|tflite [resnet50|mobilenet|ssd-mobilenet|ssd-resnet34|ssd-resnet34] [cpu|gpu]"
```

Comment:
there are 2 ssd-resnet34 here.

Reply:
removed the 2nd ssd-resnet34. The plan was that you can pick if you want the onnx model converted from pytorch or tensorflow. The later I have not uploaded to zenodo since Tom will give us a new tensorflow model anyway ... going pick this up again once the new model is there and maybe switch onnx to use the tensorflow model.